server/pyproject.toml

```
@@ -37,7 +38,7 @@ pytest = "^7.1.2"
 black = "^22.3.0"
 flake8 = "^3.8.4"
 isort = "^5.6.4"
-mypy = "0.930"
+mypy = "^0.950"
```

```suggestion
mypy = ">=0.961,<1"
```

Given mypy's versionning, `^0.961` is equivalent to `0.961` here

---

thx :)